generators/app/index.js

```diff
@@ -17,6 +16,28 @@ module.exports = generators.Base.extend({
     });
   },
 
+  prompting: function () {
+    if (!this.options.ui) {
```

Comment:
It'd be better to use inquirer `when` option to skip rather than an `if` statement. Then it'd be easier to add other prompts in the future.